rc/targets.go

```diff
@@ -40,6 +40,24 @@ type targetDetailsYAML struct {
 	Targets map[TargetName]TargetProps
 }
 
+func getFileName() (flyrc string) {
```

Comment:
this should be called something like `flyrcPath` instead

also named return values are discouraged; could just be `return filepath.Join(...)` instead anyway